### ISSUE TITLE
Confused about the introduction of Filters

### ISSUE THREAD

**Issue by watermoon** [google/leveldb#848]
Section of "Filters" in page(https://github.com/google/leveldb/blob/master/doc/index.md)

Method "CreateFilter", the last line of code
```cpp
return builtin_policy_->CreateFilter(&trimmed[i], n, dst);
```
should it changed to
```cpp
return builtin_policy_->CreateFilter(&trimmed[0], n, dst);
```
for the variable `i` is out of its scope.

**Comment by ghemawat** [comment]
On Sun, Nov 22, 2020 at 9:25 PM watermoon <notifications@github.com> wrote:

> Section of "Filters" in page(
> https://github.com/google/leveldb/blob/master/doc/index.md)
>
> Method "CreateFilter", the last line of code
>
> return builtin_policy_->CreateFilter(&trimmed[i], n, dst);
>
> should it changed to
>
> return builtin_policy_->CreateFilter(&trimmed[0], n, dst);
>
> for the variable i is out of its scope.
>

Yes, I will change it. Thanks for pointing out the problem.

**Comment by cmumford** [comment]
Fixed in 2802398c94b3b5708f111dae58ac1b738613bbf8.
